server/conf/i18n/messages

```diff
@@ -590,3 +590,11 @@ banner.httpsSectionContent=A lock ( {0} ) or https:// means you''ve safely conne
 banner.notForProductionBannerLine1=This site is for testing purposes only. Do not enter personal information.
 # A friendly message shown to the user with a call to action to go to the production site link provided in {0}
 banner.notForProductionBannerLine2=To apply to a program or service go to {0}.
+
+#------------------------------------------------------------------------------#
+# GUEST BANNER - alert explaining that user is logged in as a guest #
+#------------------------------------------------------------------------------#
+# A message in an alert about ending their session when they are finished. The placeholder in {0} will be a link with the text 'end your session'.
+banner.guestBannerText=You are applying as a guest. After you finish applying, {0} to protect your data.
+# A call to action within another message to end your guest sessiont.
```

Comment:
**suggestion (non-blocking):** May also be worth saying that this will include a link to end the session, but I guess "call to action" kind of means that

Reply:
done